Update get-started.md

sfrechette opened the pull request:
Typos "...compeition ewith differentiated.."  fix to "...competition with differentiated..."

azurecla commented:
Hi __@arkulkarni__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
@dvana 
FYI

Commit:
Update get-started.md

Typos "...compeition ewith differentiated.."  fix to "...competition with differentiated..."
